RockWeb/Blocks/Groups/GroupDetail.ascx

```diff
@@ -277,6 +277,7 @@
                             <span class="pull-right">
                                 <asp:HyperLink ID="hlAttendance" runat="server" CssClass="btn btn-sm btn-default" ToolTip="Attendance"><i class="fa fa-check-square-o"></i></asp:HyperLink>
                                 <asp:HyperLink ID="hlMap" runat="server" CssClass="btn btn-sm btn-default" ToolTip="Interactive Map"><i class="fa fa-map-marker"></i></asp:HyperLink>
+                                <asp:LinkButton ID="btnCopy" runat="server" CssClass="btn btn-default btn-sm fa fa-clone" OnClick="btnCopy_Click" ToolTip="Copy Group" />
```

Comment:
Can you change the ToolTip text to `Copies the group and all of its associated authorization rules`